common/src/test/org/apache/hadoop/hive/conf/TestHiveConf.java

```diff
@@ -252,4 +254,16 @@ public void testAdditionalConfigFiles() throws Exception{
     f2.delete();
     fileBakHiveSite.delete();
   }
+
+  @Test
+  public void testGetMatchingEntries() {
+    HiveConf conf = new HiveConf();
+    conf.setInt("hive.compactor.worker.iceberg.threads", 4);
+
+    List<Map.Entry<String, String>> entries = conf.getMatchingEntries(Constants.COMPACTION_POOLS_PATTERN);
```

Comment:
also, COMPACTION_POOLS_PATTERN is incorrect, please change to 
````
^hive\.compactor\.worker\.(.+)\.threads$
````

Reply:
Fixed